core/arch/arm/include/sm/optee_smc.h

```diff
@@ -93,6 +93,12 @@
 #define OPTEE_SMC_SHM_CACHED		1
 
 /*
+ * Privileged call flag. Used by supplicant. Asks OP-TEE to use reserved
+ * thread if all regular threads are used.
+ */
+#define OPTEE_SMC_FLAGS_PRIVILEGED	BIT(0)
```

Comment:
You mean that you don't want to allocate whole register for flags? Possibly we can pass flag in r0 then.